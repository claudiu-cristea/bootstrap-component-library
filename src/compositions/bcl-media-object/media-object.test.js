import { renderTwigFileAsNode } from "@openeuropa/bcl-test-utils";

import demoData from "@openeuropa/bcl-media-object/dataSmallUser";

const template = "@oe-bcl/bcl-media-object/media-object.html.twig";
const render = (params) => renderTwigFileAsNode(template, params);

describe("OE - Media object", () => {
  test("renders correctly", () => {
    expect.assertions(1);

    return expect(render(demoData)).resolves.toMatchSnapshot();
  });
});
