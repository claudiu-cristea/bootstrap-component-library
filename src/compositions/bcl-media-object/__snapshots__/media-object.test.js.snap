// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`OE - Media object renders correctly 1`] = `
<jest>
  <div
    class="align-items-center mb-3 d-flex"
  >
    <div
      class="flex-shrink-0"
    >
      <img
        alt="Stefan Mayer"
        class="rounded-pill"
        src="https://picsum.photos/50/50?random=1"
      />
    </div>
    <div
      class="flex-grow-1 ms-3"
    >
      <a
        class="fs-5 text-decoration-none d-block mb-0"
        href="example.html"
      >
        Stefan Mayer
      </a>
      <span
        class="text-muted me-4-5"
      >
        DG Test
      </span>
      <span
        class="text-muted me-4-5"
      >
        Manager
      </span>
    </div>
  </div>
</jest>
`;
